packages/fake-backends/src/FakeChatClient.ts

```diff
@@ -74,8 +74,9 @@ export class FakeChatClient implements IChatClient {
         ...baseChatThreadEvent(thread),
         createdOn: now,
         properties: {
-          topic: request.topic
-          // metadata: {}
+          topic: request.topic,
+          /* @conditional-compile-remove(signaling-beta) */
```

Comment:
remove all these conditional compile line

Reply:
these are needed, the signalling package doesn't have that metadata element that is added back

Reply:
Oh of course this is signalling not common - my bad!